packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -4,6 +4,19 @@ import {omit, has} from 'lodash';
 import {shallowEqual, getDisplayName} from './utils';
 
 /**
+ * @typedef {object} ConnectorDescription
```

Comment:
I started exploring the possibility of writing the <Configure /> widget, and didn't have a proper documentation for the object that could be passed to `createConnector`. This is its documentation, and I thought maybe we could keep it since it was already done.